collect_app/src/main/java/org/odk/collect/android/utilities/MediaUtils.java

```diff
@@ -617,9 +617,8 @@ public static boolean isGooglePhotosUri(Uri uri) {
      * @param uri The Uri to check
      * @return Whether the Uri authority is Google Drive.
      */
-    public static boolean isGoogleDriveDocument(Uri uri) {
-        return "com.google.android.apps.docs.storage".equals(uri.getAuthority())
-                || uri.getAuthority().startsWith("com.google.android.apps.photos.content");
+    private static boolean isGoogleDriveDocument(Uri uri) {
+        return uri.getAuthority().startsWith("com.google.android.apps.");
```

Comment:
It sounds like the safest fix here would be to check for `com.google.android.apps.docs.storage` AND `"com.google.android.apps.photos.content`. Why did you drop the photos URI?

Reply:
I know GD uri might contain:
`com.google.android.apps.docs.storage`
`com.google.android.apps.docs.storage.legacy`
`com.google.android.apps.photos.content`

all of them starts with `com.google.android.apps.` so I think it's ok to use it, maybe there is another case we don't know like this issue.

Reply:
if it starts with `com.google.android.apps.` it msut be GD uri no matter what is after that las dot.

Reply:
I'd rather be conservative and do a whitelist. That is, there might also be `com.google.android.apps.bad.stuff` with unknown behavior.

Reply:
Yeah, that would be ideal.

Reply:
Ok, it's done.